scripts/update-lobster.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+if [ -n "$LOBSTER_TEMP_DIR" ] && [ -n "$LOBSTER_TEMP_DIR" ]; then
+  set -o errexit
+  set -o xtrace
+
+  echo $LOBSTER_TEMP_DIR
+  LOBSTER_REPO=https://github.com/evergreen-ci/lobster.git
+  LOBSTER_TEMP_ASSETS_DIR="$LOBSTER_TEMP_DIR/build"
+  LOBSTER_STATIC_DIR="$LOBSTER_TEMP_ASSETS_DIR/static"
+  LOBSTER_HTML="$LOBSTER_TEMP_ASSETS_DIR/index.html"
+  EVG_LOB_MANIFEST_DIR="$EVGHOME/public/static/lobster"
+  EVG_LOB_STATIC_DIR="$EVG_LOB_MANIFEST_DIR/static"
+  EVERGREEN_TEMPLATES_DIR="$EVGHOME/service/templates"
+
+  # clone lobster repo
+  mkdir -p $LOBSTER_TEMP_DIR
+  git clone $LOBSTER_REPO $LOBSTER_TEMP_DIR
+
+  # build lobster
+  cd $LOBSTER_TEMP_DIR
+  npm install
+  # env variables used during lobster build step
+  export PUBLIC_URL=/static/lobster
+  export REACT_APP_LOGKEEPER_BASE=https://logkeeper.mongodb.org
+  export REACT_APP_EVERGREEN_BASE=https://evergreen.mongodb.com
+  #
+  npm run build 
+
+  # delete lobster static files in evergreen repo
+  rm -rf $EVG_LOB_STATIC_DIR
```

Comment:
We need to overwrite old dist files with new ones. I'm unsure how to avoid using the `rm -rf` command here

Reply:
Hmm I guess it's alright if we have to modify the JS files.